.gitignore

```diff
@@ -11,3 +11,4 @@ logs/
 *.tar
 .gdb_history
 .DS_Store
+.vscode
```

Comment:
Probably not needed. You can configure a global gitignore instead: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Fixed